README.md

```diff
@@ -1,86 +1,126 @@
 
-The .NET Framework 4.6.2 is included in the [Windows Server Core base image](https://hub.docker.com/r/microsoft/windowsservercore/). You can pull/FROM microsoft/windowsservercore to use .NET Framework 4.6.2.
+# What is the .NET Framework?
 
-To make discovery easier, we have also provided the same image (with a FROM wrapper) in this repo, enabling you to pull `microsoft/dotnet-framework`.
+The .NET Framework is a general purpose development platform maintained by Microsoft. It is the most popular way to build client and server applications for Windows and Windows Server. It is included with Windows, Windows Server and Windows Server Core. It includes server technologies such as ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF) applications, which you can run in Docker containers.
 
-- Optional tags: `latest`, `4.6.2`
-- Dockerfile: [4.6.2/Dockerfile](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.6.2/Dockerfile)
+.NET has several capabilities that make development easier, including automatic memory management, (runtime) generic types, reflection, asynchrony, concurrency, and native interop. Millions of developers take advantage of these capabilities to efficiently build high-quality web and client applications.
 
-## .NET Framework 3.5
+You can use C#, F# and VB to write .NET Framework apps. C# is simple, powerful, type-safe, and object-oriented while retaining the expressiveness and elegance of C-style languages. F# is a multi-paradigm programming language, enabling both functional and object-oriented patterns and practices. VB is a rapid development programming language with the deepest integration between the language and Visual Studio, providing the fastest path to a working app.   
 
-.NET Framework 3.5 is an optional component in Windows Server Core. The official Docker image for the .NET Framework 3.5 is included in this Docker repository and is in preview. You can pull/FROM microsoft/dotnet-framework:3.5 to use .NET Framework 3.5.
+The .NET Framework was first released by Microsoft in 2001. The latest version is .NET Framework 4.6.2.
 
-- Required tag: `3.5`
-- Dockerfile: [3.5/Dockerfile](https://github.com/Microsoft/dotnet-framework-docker/blob/master/3.5/Dockerfile)
+> https://docs.microsoft.com/dotnet/articles/framework/
 
-## .NET Framework 4.6 and 3.5 and ASP.NET
+![logo](https://avatars0.githubusercontent.com/u/9141961?v=3&amp;s=100)
 
-You should use the ASP.NET Docker images if you want to package a .NET Framework ASP.NET app with Docker. The following Docker images are available in the [microsoft/aspnet](https://hub.docker.com/r/microsoft/aspnet/) Dockerhub repo.
+# How to use these Images
 
-- microsoft/aspnet:4.6.2 - For .NET Framework 4.x ASP.NET applications.
-- microsoft/aspnet:3.5 - For .NET Framework 3.5 ASP.NET applications.
+## Deploying a .NET Framework 4.x application with Docker
```

Comment:
I feel like it would be worth mentioning and referencing Windows Containers at this point.  I want to make sure noobs try this out with the correct Docker configuration.